packages/apex-node/src/utils/traceFlags.ts

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import { Connection } from '@salesforce/core';
+import * as util from 'util';
+import { nls } from '../i18n';
+import { DEFAULT_DEBUG_LEVEL_NAME, LOG_TYPE } from '../logs/constants';
+
+interface UserRecord {
+  Id: string;
+}
+
+interface DebugLevelRecord {
+  ApexCode: string;
+  VisualForce: string;
+}
+
+interface TraceFlagRecord {
+  Id: string;
+  LogType: string;
+  DebugLevelId: string;
+  StartDate: Date | undefined;
+  ExpirationDate: Date | undefined;
+  DebugLevel: DebugLevelRecord;
+}
+
+interface DataRecordResult {
+  id?: string;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  errors?: any[];
```

Comment:
can we specify a type for the errors here?

Reply:
@rcoringrato-sfdc any thoughts on if we can specify type? I see in the original code that it's the same as it is here: https://github.com/forcedotcom/salesforcedx-vscode/blob/7d089a2cee49b5989066d76f55932b6a35f581a5/packages/salesforcedx-vscode-apex-replay-debugger/src/commands/traceFlags.ts#L30